### DISCUSSION TITLE
Import subscription

### DISCUSSION THREAD

**Discussion by mjain234** [libre-tube/LibreTube#478] [General]
I am trying to import subscription for libretube following this process

1. Go to the Google takeout manager.
2. Log in if asked.
3. Click on "All YouTube data included", then on "Deselect all", then select only "subscriptions" and click "OK".
4. Click on "Next step" and then on "Create export".
5. Click on the "Download" button after it appears.

Is this correct or is there any other way , please help!

**Comment by Bnyro** [comment]
should be fine

**Comment by xrat** [comment]
FWIW, I've been following these steps many times but my takeout ZIP archives mention "YouTube and YouTube Music" but no "Subscriptions". I've never managed to export my 200+ subscriptions, and I've no idea why.

**Reply by Bnyro** [reply]
There must be a file called subscriptions (or subscriptions translated into the language you use) inside the zip when you selected YouTube at the Google Taktout.

**Reply by xrat** [reply]
Right, but it's missing.

**Reply by Bnyro** [reply]
Then Google doesn't want you to leave :(

**Reply by Bnyro** [reply]
You exported all your YouTube data?

**Reply by xrat** [reply]
Well, I am using alternatives already and I am about to add my subscriptions to LibreTube. Luckily I got a YT export from 2020 or so when it was still possible on the main frontend. But it would be way more comfortable if I could export an up-to-date list. And, yes, I tried it either way. If I export all data I get 1 song I purchased via YT, but no history, no subscriptions. If I choose to export only subscriptions, the takeout ZIP has not "youtube" directory at all.

**Reply by xrat** [reply]
**I succeeded**, finally! Thanks @Bnyro , somehow you kept me going looking for why it didn't work!
So, the reason was that my YT account is rather old. It got an automatically generated eMail assigned (`@pages.plusgoogle.com`) and was somehow linked to my main Google account. Anyway, at https://takeout.google.com/ I needed to switch to my YT "account" by clicking the avatar in the top right. There, my YT account is listed as "Brand account". Once chosen, I could proceed as usual.
